src/ResultRenderer/OutputFailureRenderer.php

```diff
@@ -32,11 +32,13 @@ public function __construct(StdOutFailure $result)
     public function render(ResultsRenderer $renderer)
     {
         return sprintf(
-            "  %s\n%s\n\n  %s\n%s\n",
+            "  %s\n%s\n\n  %s\n%s\n" . ($this->result->getWarnings() ? "\n  %s\n%s\n" : ""),
```

Comment:
I think we can add a method to the `Result` like `hasWarnings()` - it's a bit more expressive.